shopelectro/tests/tests_models.py

```diff
@@ -86,3 +86,32 @@ def test_get_brands_from_products(self):
                 p: p.tags.filter(group__name=settings.BRAND_TAG_GROUP_NAME).first()
                 for p in products
             }
+
+    def test_get_catalog_blocks(self):
```

Comment:
```suggestion
    def test_get_catalog_blocks_queries(self):
```

The name will able us to get rid from pydoc. It just doubles code lines now

Reply:
It is already in a relevant class `QueryQuantities`, so I'll remove the comment